test/common/performance/sender/http_load_generator.go

```diff
@@ -63,20 +68,26 @@ func NewCloudEventsTargeter(sinkUrl string, msgSize uint, eventType string, even
 }
 
 func (cet CloudEventsTargeter) VegetaTargeter() vegeta.Targeter {
+	uuidGen := fastuuid.MustNewGenerator()
+
+	ceType := []string{cet.eventType}
+	ceSource := []string{cet.eventSource}
+	ceSpecVersion := []string{"0.2"}
+	ceContentType := []string{"application/json"}
+
 	return func(t *vegeta.Target) error {
 		t.Method = http.MethodPost
 		t.URL = cet.sinkUrl
 
-		t.Header = make(http.Header)
-
-		t.Header.Set("Ce-Id", uuid.New().String())
-		t.Header.Set("Ce-Type", cet.eventType)
-		t.Header.Set("Ce-Source", cet.eventSource)
-		t.Header.Set("Ce-Specversion", "0.2")
+		t.Header = make(http.Header, 5)
 
-		t.Header.Set("Content-Type", "application/json")
+		t.Header["Ce-Id"] = []string{uuidGen.Hex128()}
 
-		t.Body = []byte("\"" + generateRandString(cet.msgSize) + "\"")
+		t.Header["Ce-Type"] = ceType
+		t.Header["Ce-Source"] = ceSource
+		t.Header["Ce-Specversion"] = ceSpecVersion
+		t.Header["Content-Type"] = ceContentType
+		t.Body = generateRandStringPayload(cet.msgSize)
```

Comment:
(Replying to https://github.com/knative/eventing/pull/2195#discussion_r348373119). I'm fine with either strategy as long as we're aware of the tradeoffs (and sounds like we are). If you want to move it back to where it was for better load generator performance, that's fine. Making it configurable would be great but doesn't have to happen in this PR.

Reply:
https://github.com/knative/eventing/issues/2208